doc/api_extensions.md

```diff
@@ -74,3 +74,6 @@ flag to each XHR Request).
 Some browsers like Firefox and Safari can't accept server response without
 `Access-Control-Allow-Credentials: true` header. To ensure that the server will
 return a response with that header, set `core.https_allowed_credentials=true`.
+
+##image compression
+Add support to specify compression algorithm when publishing an image
```

Comment:
This should mention that this is the compression_algorithm property on POST /1.0/images